docs/example-scenario/apps/sap-dev-test.md

```diff
@@ -0,0 +1,109 @@
+---
+
+# SAP for a dev/test workload
+
+This architectural design shows a set of proven practices for running SAP NetWeaver in a Windows or Linux environment on Azure. The database is AnyDB, the SAP term for any supported DBMS (that isn't SAP HANA). This architecture is deployed with specific virtual machine (VM) sizes that can be changed to accommodate your organization's needs.
+
+## Related use cases
+
+Consider this scenario for the following use cases:
+
+* Non-Critical SAP non-productive workloads (Sandbox, Development, Test, Quality Assurance)
+* Non-critical SAP Business One workloads
+
+## Architecture
+
+![Diagram](media/sap-2tier/SAP-Infra-2Tier_finalversion.png)
+
+This scenario covers the provision of a single SAP system database and SAP application Server on a single virtual machine, the data flows through the scenario as follows:
+
+1. Customers from the Presentation Tier use their SAP gui, or other user interfaces (Internet Explorer, Excel or other web application) on premise to access the Azure based SAP system.
+2. Connectivity is provided through the use of the established Express Route. The Express Route is terminated in Azure at the Express Route Gateway. Network traffic routes through the Express Route gateway to the Gateway Subnet and from the gateway subnet to the Application Tier Spoke subnet and via a Network Security Gateway to the SAP application virtual machine.
+3. The identity management servers provide authentication services.
+4. The Jump Box provides local management capabilities.
+
+### Components
+
+* [Resource Groups](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview#resource-groups) is a logical container for Azure resources.
+* [Virtual Networks](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-overview) is the basis of network communications within Azure
+* [Virtual Machine](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/overview) Azure Virtual Machines provides on-demand, high-scale, secure, virtualized infrastructure using Windows or Linux Server
+* [Express Route](https://docs.microsoft.com/en-us/azure/expressroute/expressroute-introduction) lets you extend your on-premises networks into the Microsoft cloud over a private connection facilitated by a connectivity provider.
+* [Network Security Group](https://docs.microsoft.com/en-us/azure/virtual-network/security-overview) lets you limit network traffic to resources in a virtual network. A network security group contains a list of security rules that allow or deny inbound or outbound network traffic based on source or destination IP address, port, and protocol. 
+
+### Alternatives
+
+* List of alternative options and why you might use them.
+
+## Considerations
+
+### Availability
+
+For any Single Instance Virtual Machine using premium storage for all Operating System Disks and Data Disks, we guarantee you will have Virtual Machine Connectivity of at least 99.9%.
```

Comment:
We should avoid giving any SLA numbers and refer to the official SLA doc. In other articles, we say "Microsoft offers a service level agreement (SLA) for single VM instances."

Reply:
fixed